app/models/event.rb

```diff
@@ -193,6 +193,14 @@ def compute_occupied_places
     application_letters.where(status: ApplicationLetter.statuses[:accepted]).count
   end
 
+  # Returns whether there are rejected applications with no status notification sent of the event
+  #
+  # @param none
+  # @return [Boolean] true if there are rejected participants without status notification sent
+  def has_rejected_participants_without_status_notification?
+    application_letters.where(status: ApplicationLetter.statuses[:rejected], status_notification_sent: false).count > 0
```

Comment:
I think you can use [exists?](http://apidock.com/rails/ActiveRecord/Base/exists%3F/class). I think it is a little bit better to read and with many applications_letters it is lots faster.